documentation-website/Writerside/snippets/exposed-dao/src/main/kotlin/org/example/entities/CityEntity.kt

```diff
@@ -0,0 +1,12 @@
+package org.example.entities
+
+import org.example.tables.CitiesTable
+import org.jetbrains.exposed.dao.ImmutableEntityClass
+import org.jetbrains.exposed.dao.IntEntity
+import org.jetbrains.exposed.dao.id.EntityID
+
+class CityEntity(id: EntityID<Int>) : IntEntity(id) {
+    val name by CitiesTable.name
+
+    companion object : ImmutableEntityClass<City>(CitiesTable)
+}
```

Comment:
Does this compile ok? `ImmutableEntityClass` should require 2 type parameters and `City` doesn't seem to appear in the snippets.
```kt
companion object : ImmutableEntityClass<Int, CityEntity>(CitiesTable)
```